src/index.ts

```diff
@@ -41,10 +41,11 @@ export default function useOnClickOutside(
     }
 
     const listener = (event: PossibleEvent) => {
+      const element = ref instanceof HTMLElement || ref == null ? ref : ref.current;
```

Comment:
```suggestion
      const element = !ref || 'onblur' in ref ? ref : ref.current;
```

`instanceof` is not safe cross-realm, so I generally prefer to do checks avoiding its usage